app/scrapers/generalized.py

```diff
@@ -84,3 +84,34 @@ def search_without_count(self, query):
         soup = BeautifulSoup(response.text, 'html.parser')
         urls = self.parse_response(soup)
         return urls
+
+    def video_search_without_count(self, query):
+        """
+            Search for the query and return set of urls
+            Returns: list
+        """
+        urls = []
+        if self.name in ['parsijoo']:
+            url = self.videoURL
+            payload = {self.queryKey: query}
+        response = requests.get(url, headers=self.headers, params=payload)
+        soup = BeautifulSoup(response.text, 'html.parser')
+        urls = self.parse_video_response(soup)
+        if urls == []:
+            return "No video with this Keyword"
+        else:
+            return urls
```

Comment:
Could we just do __return urls or "No video with this Keyword"__ because all other falsey values are just as dangerous as an empty list.